src/WCSimPMTObject.cc

```diff
@@ -1222,6 +1222,7 @@ float BoxandLine20inchHQE::HitTimeSmearing(float Q) {
 
 G4float* BoxandLine20inchHQE::Getqpe()
 {
+  
```

Comment:
Take out this extra line so WCSimPMTObject doesn't show up in the diff for this PR.